Dockerfile

```diff
@@ -1,81 +1,34 @@
-#     ú   [Type        : Dockerfile....] [Rating           : ........]   ú
-#
-#
-#     ú     Well, this is a little Dockerfile that have many functions   ú
-#           for quantifying phenotypes...enjoy....                       ú
-#
-#     ú                                                                  ú
-#     ³                                                                  ³
-#     ³                                                                  ³
-#     ³                                                                  ³
-#     ³                                                                  ³
-#     º                                                                  º
-#     ÈÍÍÍÄÄÄÄÄÄÄÄÄÄÄÄÄÄ Ä  ÄÄ  Ä  Ä   ú   Ä  Ä  ÄÄ  Ä ÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÍÍÍ¼
-#
-#     Greets: ...
-#
-#
-#                                 - [ BROAD‘15 ] -
-#                                                          -0x00B1 [05/06/84]
-FROM ubuntu:14.04
-RUN apt-get -y update                                            && \
-    apt-get -y upgrade                                           && \
-    apt-get -y install                                              \
-      cython                                                        \
-      git                                                           \
-      openjdk-7-jdk                                                 \
-      python-h5py                                                   \
-      python-imaging                                                \
-      python-libtiff                                                \
-      python-lxml                                                   \
-      python-matplotlib                                             \
-      python-mysqldb                                                \
-      python-numpy                                                  \
-      python-pandas                                                 \
-      python-pip                                                    \
-      python-scipy                                                  \
-      python-skimage                                                \
-      python-sklearn                                                \
-      python-vigra                                                  \
-      python-wxgtk2.8                                               \
-      python-zmq                                                    \
-      xvfb
+FROM ubuntu:16.04
+# Install CellProfiler dependencies
+RUN   apt-get -y update &&                                          \
+      apt-get -y install                                            \
+        build-essential    \
+        cython             \
+        git                \
+        libmysqlclient-dev \
+        libhdf5-dev        \
+        libxml2-dev        \
+        libxslt1-dev       \
+        openjdk-8-jdk      \
+        python-dev         \
+        python-pip         \
+        python-h5py        \
```

Comment:
Let’s use the versions of H5Py, matplotlib, SciPy, and NumPy available from PyPI.